recipes/brass/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+INST_PATH="."
+
+git clone git://git.code.sf.net/p/pstreams/code c++/pstreams
```

Comment:
This does not look very reproducible, can you create a separate package for it and fixing the version?